sdk/spring/azure-spring-boot-test-aad-resource-server-by-filter/src/test/java/com/azure/test/aad/filter/group/count/AADGroupsCountIT.java

```diff
@@ -44,7 +44,12 @@ public void beforeAll() {
         properties.put("azure.activedirectory.tenant-id", AAD_TENANT_ID_1);
         properties.put("azure.activedirectory.client-id", AAD_MULTI_TENANT_CLIENT_ID);
         properties.put("azure.activedirectory.client-secret", AAD_MULTI_TENANT_CLIENT_SECRET);
-        properties.put("azure.activedirectory.user-group.allowed-groups", "group1,group2");
```

Comment:
if the old properties is still supported, maybe we should add new test instead of modify the current one?

Reply:
This change has been revert. New UT is the *AADAccessTokenGroupRolesExtractionTest*.